tools/rosmsg/test/test_rosmsg_command_line.py

```diff
@@ -54,24 +54,24 @@ def test_cmd_help(self):
         
         for cmd in ['rosmsg', 'rossrv']:
             glob_cmd=[os.path.join(_SCRIPT_FOLDER, cmd)]
-            output = Popen(glob_cmd, stdout=PIPE, env=self.new_environ).communicate()[0]
+            output = Popen(glob_cmd, stdout=PIPE, env=self.new_environ).communicate()[0].decode()
```

Comment:
Just noting an alternative to decoding is to use "text mode" which makes `communicate()` return two strings. It can be done with the keyword arg `Popen(..., text=true)` in python 3.7+

Reply:
A Python 3.7 only feature couldn't land on the `melodic-devel` branch which targets Ubuntu Bionic.

Reply:
Roger. The [docs](https://docs.python.org/3/library/subprocess.html#subprocess.Popen.communicate) say `text=True` is an alias for `universal_newlines=True`. `text` was added in 3.7, but it looks like `universal_newlines` has been around from the start.